packages/shared-ux/page/analytics_no_data/src/services.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useContext } from 'react';
+import { Observable } from 'rxjs';
+
+type DataView = unknown;
+
+interface DataViewEditorOptions {
+  onSave: (dataView: DataView) => void;
+}
+
+/**
+ *
+ */
+export interface Services {
+  addBasePath: (url: string) => string;
+  canAccessFleet: boolean;
+  canCreateNewDataView: boolean;
+  currentAppId$: Observable<string | undefined>;
+  dataViewsDocLink: string;
+  hasDataView: () => Promise<boolean>;
+  hasESData: () => Promise<boolean>;
+  hasUserDataView: () => Promise<boolean>;
+  kibanaGuideDocLink: string;
+  navigateToUrl: (url: string) => Promise<void>;
+  openDataViewEditor: (options: DataViewEditorOptions) => () => void;
```

Comment:
We could probably use here the exposed interface to open the editor to keep in sync

Reply:
Let me be clear: we can't use the interface provided by the `data_view_editor` plugin.  Packages cannot access plugins, (but that may have changed recently with the switch to "plugins are packages" move).  @spalger ?

Reply:
Nope, plugins are synthetic packages, they don't participate in Bazel just have module ids so we can move closer to that being the case but each plugin will have to be migrated so you'll know when we're close.